src/dbg/debugger_commands.cpp

```diff
@@ -142,6 +142,11 @@ CMDRESULT cbDebugRun(int argc, char* argv[])
     // Don't "run" twice if the program is already running
     if(dbgisrunning())
         return STATUS_ERROR;
+    if(!ThreaderIsAnyThreadActive())

```

Comment:
Oh, if I create a thread and then suspend the main thread and run, TitanEngine's `ThreaderIsAnyThreadActive` still returns false. This code has to be removed or TitanEngine has to be fixed.